editor/plugins/script_text_editor.h

```diff
@@ -109,6 +112,14 @@ class ScriptTextEditor : public ScriptEditorBase {
 	void _goto_line(int p_line) { goto_line(p_line); }
 	void _lookup_symbol(const String &p_symbol, int p_row, int p_column);
 
+	enum CONVERT_CASE
+	{
+		UPPER,
+		LOWER,
+		CAPITALIZE,
+	};
+	void _convert_case(int p_case);
```

Comment:
Then it would be easier to read too if you use the enum in the signature, `void _convert_case(ConvertCase p_case);`. And actually maybe rename the scope to something like `CaseStyle` which would be more explicit.